requirements.md

```diff
@@ -83,4 +93,4 @@ Applications not maintained by Anbox Cloud may have different hardware recommend
  - **etcd**: https://etcd.io/docs/v3.4.0/op-guide/hardware/
  - **HAProxy** (load balancer for the Stream Gateway and the dashboard): https://www.haproxy.com/documentation/hapee/latest/installation/getting-started/os-hardware/#hardware-requirements
  
-Please note that these are just baselines and should be adapted to your worload. No matter the application, [measuring performances](https://anbox-cloud.io/docs/monitoring/monitoring-installation) is always important.
\ No newline at end of file
+Please note that these are just baselines and should be adapted to your worload. No matter the application, [measuring performances](https://anbox-cloud.io/docs/monitoring/monitoring-installation) is always important.
```

Comment:
```suggestion
Please note that these are just baselines and should be adapted to your workload. No matter the application, [measuring performances](https://anbox-cloud.io/docs/monitoring/monitoring-installation) is always important.
```

Reply:
Just saw this text hasn't changed. Let's ignore for now, I'll fix it with other typos in the docs. :)